cloud/storage/core/tools/common/python/daemon.py

```diff
@@ -38,9 +38,17 @@ def __init__(self, commands, cwd, restart_allowed=False,
         self.__ping_attempts = ping_attempts
         self.__core_pattern = core_pattern
 
-        self.__stdin_file = tempfile.NamedTemporaryFile(dir=self.__cwd, prefix="stdin_", delete=False)
-        self.__stdout_file = tempfile.NamedTemporaryFile(dir=self.__cwd, prefix="stdout_", delete=False)
-        self.__stderr_file = tempfile.NamedTemporaryFile(dir=self.__cwd, prefix="stderr_", delete=False)
+        stdin_prefix = "stdin_"
+        stdout_prefix = "stdout_"
+        stderr_prefix = "stderr_"
+        if service_name is not None:
+            stdin_prefix = "{}_{}".format(service_name, stdin_prefix)
```

Comment:
Use f-string for consistency.

Reply:
Only format function is used in this file (and lots of others)